alerts/apache/high-request-rate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "displayName": "Apache - high-request-rate",
+  "documentation": {
+    "content": "The request rate is derived from the requests metrics taken as a rate of every 5 minutes. This value should be monitored beforehand to understand what qualifies as a normal request rate so a threshold can be established. When the request rate is above the established threshold, then that means there is a large spike in traffic which logs can help diagnose if these are nefarious requests being made.",
+    "mimeType": "text/markdown"
+  },
+  "userLabels": {},
+  "conditions": [
+    {
+      "displayName": "VM Instance - workload/apache.requests",
+      "conditionThreshold": {
+        "aggregations": [
+          {
+            "alignmentPeriod": "300s",
+            "perSeriesAligner": "ALIGN_RATE"
+          }
+        ],
+        "comparison": "COMPARISON_GT",
+        "duration": "0s",
+        "filter": "resource.type = \"gce_instance\" AND metric.type = \"workload.googleapis.com/apache.requests\"",
+        "thresholdValue": 100,
+        "trigger": {
+          "count": 1
+        }
+      }
+    }
+  ],
+  "alertStrategy": {
+    "autoClose": "604800s",
+    "notificationPrompts": [
```

Comment:
Property "notificationPrompts" does not exist in object specification.

Reply:
updated